inst/include/Rcpp/date_datetime/Date.h

```diff
@@ -102,6 +102,16 @@ namespace Rcpp {
             return m_d;
         }
 
+        inline std::string format(const char *fmt = "%Y-%m-%d") const {
+            char txt[32];
+            struct tm temp = m_tm;
+            temp.tm_year -= baseYear();    // adjust for fact that system has year rel. to 1900
+            ::strftime(txt, 31, fmt, &temp);
```

Comment:
Should we check the return value of `::strftime()` here? What happens to `txt` if the user passes in a `fmt` string not recognized by `::strftime()`?